tests/phpunit/BltProject/FixTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Acquia\Blt\Tests\BltProject;
+
+use Acquia\Blt\Tests\BltProjectTestBase;
+
+/**
+ * Class FixTasksTest.
+ *
+ * Verifies that fix related tasks work as expected.
+ */
+class FixTasksTest extends BltProjectTestBase {
+
+  private $returnVar;
+  private $output = [];
+
+  /**
+   * Tests for existence of phpcbf script.
+   */
+  public function testFixPhpcbfExists() {
+    $this->executePhpcbfFileListScript();
+    $this->assertNotContains("PHP Code Beautifier was not found in this project's bin directory. Please run composer install.", $this->output);
+    $this->assertEquals(1, $this->returnVar);
+  }
+
+  /**
+   * Tests the handling of a missing file parameter.
+   */
+  public function testFixPhpcbfExitsWhenMissingArguments() {
+    $this->executePhpcbfFileListScript();
+    $this->assertContains("Missing file list parameter.", $this->output);
+    $this->assertEquals(1, $this->returnVar);
+  }
+
+  /**
+   * Tests that a file list is generated.
+   */
+  public function testFixPhpcbfWithArguments() {
+    $report = $this->createTestReport();
```

Comment:
@dooleymatt I'm confused: Don't you want to be creating a test file rather than a test report? A PHP file with standards violations in it so you can assert that PHPCBF correctly finds it?